### DISCUSSION TITLE
To add package Robust

### DISCUSSION THREAD

**Discussion by RafaelFrance** [IDEMSInternational/R-Instat#9803] [Ideas]
Hello,

First, sorry for my English.
Second, thank a lot for this great software. I am learning statistic and its help me a lot.

I would like to add into the Wish list some features of the package Robust.

I think first about :
- Trimmed Mean
- Winsorized Mean
- M-Estimator.
Ideally, this could be located in the summary menus

Thanks again

**Comment by lilyclements** [comment]
@rdstern do we offer trimmed mean, winsorised mean, or m-estimator mean in R-Instat? This should hopefully not be difficult to add in

**Comment by rdstern** [comment]
@lilyclements see below, we have trimmed mean, but that's not from the Robust package.  I think we have the robust package - partly so we could get the statistics for the skew boxplot.  So, as you say, it should be easy to add more.  Possibly re-organise this "page".

<img width="640" height="729" alt="image" src="https://github.com/user-attachments/assets/5d460018-897c-48df-a0d2-110a1c1272cc" />
